plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/cleaner/DockerAbandonedResourcesCleaner.java

```diff
@@ -68,6 +68,10 @@
   private final WorkspaceRuntimes runtimes;
   private final Set<String> additionalNetworks;
 
+  @Inject(optional = true)
```

Comment:
Can you elaborate on why have you decided to make it optional? 
I tend to avoid optional injection if possible and won't need workarounds.